x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/find/transforms/transform_find_rules_response/v1.ts

```diff
@@ -21,7 +21,8 @@ import {
 
 export const transformPartialRule = <Params extends RuleParams = never>(
   rule: Partial<Rule<Params>>,
-  fields?: string[]
+  fields?: string[],
+  isInternal: boolean = false
```

Comment:
What is the purpose of this isInternal boolean. Is it necessary?

Reply:
Maybe this should be `includeArtifacts` rather than `isInternal` to be more descriptive.